### ISSUE TITLE
[Bug]: 10.6 AdminSessionAuthenticator broken

### ISSUE THREAD

**Issue by dpfaffenbauer** [pimcore/pimcore#15671]
AdminSessionAuthenticator keeps getting called on every request that is made when logged in. This triggers the Login Throttling and therefore some requests might failed with "too many login attempts". 

In Pimcore 11, this was changed to use normal form_login, I couldn't find the PR though and I am not sure if this can be fixed in a minor release.

Our solution was to increase the rate limit for the login, but this isn't the best solution to the problem.

**Comment by dpfaffenbauer** [comment]
Basically this line here is at fault: https://github.com/pimcore/pimcore/blob/10.6/bundles/AdminBundle/Security/Authenticator/AdminSessionAuthenticator.php#L44

It starts the authentication process, triggering the Symfony LoginThrottleListener

**Comment by dpfaffenbauer** [comment]
After some further investigation: The system in Pimcore 10.5/10.6 has been implemented completely wrong. You are not supposed to authenticate every single request, you should create a token and be done with it. Since Pimcore 10 uses it's own Session, I guess that is the reason for this implementation.

**Comment by dvesh3** [comment]
@dpfaffenbauer could you please share a reproducible case or setup? we'll take a look. thanks!

**Comment by dpfaffenbauer** [comment]
@dvesh3 every Pimcore 10.5 or 10.6 Setup with the new authenticator manager ;). It is not a 100% reproducible every time. It's easier to reproduce if you set login throttling to 1

```
security:
...
    firewalls:
...
        pimcore_admin:
...
            login_throttling:
                max_attempts: 1
...
```

**Comment by dpfaffenbauer** [comment]
@dvesh3 I am happy to help in any way, just let me know.

**Comment by dvesh3** [comment]
@dpfaffenbauer It seems the authenticator has been like this only since v4 as there we also called [authenticator ](https://github.com/pimcore/pimcore/blob/v6.9.6/bundles/AdminBundle/Security/Guard/AdminAuthenticator.php#L117) in each request for the reason it being [`stateless`](https://github.com/pimcore/pimcore/blob/c1791bb2344395d3b29920ac62bc079592964cde/.github/ci/files/config/packages/security.yaml#L24). I don't think so we can change the behavior without refactoring and rewriting the authentication which might then requires `security.yaml` to change leading to breaking changes. Do you have any suggestion to solve this without breaking change?

Also, I'm not able to reproduce the "too many login attempts" issue even with `max_attempts: 1` config.

**Comment by dpfaffenbauer** [comment]
the request is not stateless, it still uses the session to authenticate ;). But with the old authentication system, pimcore also implemented the Login throttling itself. Symfony now does it differently now and breaks things.

I am honestly not sure how to fix it properly. We currently cannot update to Pimcore 11 due some other dependencies not being ready yet. We currently just increased the throttling limit, but from a security perspective, that is absolutely the worst solution.

I could replicate it locally with max_attempts 1, maybe you need a faster machine? 🤣  I can show you if you want to see it 😉

**Comment by dvesh3** [comment]
JFYI, 10.6 reaching it's EOL soon so this will not be something we want to refactor at this moment now and this is already fixed on v11. 

> I could replicate it locally with max_attempts 1, maybe you need a faster machine? 🤣 I can show you if you want to see it 😉

Haha, my machine doesn't take [5 mins](https://github.com/pimcore/demo/blob/10.3/config/packages/security.yaml#L38) to hit each request and "too many login attempts" is thrown when authentication **fails** on multiple tries so you might want to look into your setup.

**Comment by dpfaffenbauer** [comment]
`max_attempts` in `5 minutes` means that if one login fails within 5 minutes, your are blocked. And since requests happen in parallel, where the login didn't happen yet fully, the rate limiter just blocks the request. It's a bit hard to explain if you haven't debugged it ;).

I will try to make it reproduceable on a standard Pimcore installation and create a repo with it, maybe then it helps. For the Community Edition, it is too late for a prober fix anyway, I am more thinking about LTS here. It is a crucial security problem.

**Comment by dvesh3** [comment]
Fixed by #15718
